### PR TITLE
More commands

### PR DESCRIPTION
**Pull request by lucasMarioza**

### CODE REVIEW

File: bot-get.js [hunk 1]
```diff
@@ -31,4 +31,124 @@ There will be multiple "regions" and "locations", but the Progression region wil
...
+
+function onEvent(message) {
+    chatipelago.maybeTriggerItemLocationMap();
+    switch (message) {
+        /*case '/table':
+            attemptClaimTable();
+            break;
+        case '/button':
+            attemptPressButton();
+            break;*/
```
Comment (Dranzior) [inline bot-get.js:54]: We can probably erase these code snipped and not just comment them out at this point

File: bot-get.js [hunk 2]
```diff
@@ -31,4 +31,124 @@ There will be multiple "regions" and "locations", but the Progression region wil
...
 
-*/
\ No newline at end of file
+*/
+
+var goal = false;
+
+var server = require('./server.js');
+var webhook = require('./webhook-put.js');
+var chatipelago = require('./archipelagoHelper.js');
```
Comment (Dranzior) [inline bot-get.js:40]: this variable named come from how 1 had 1st name the helper file, before renaming it, maybe rename the variable too

File: bot-get.js [hunk 3]
```diff
@@ -31,4 +31,124 @@ There will be multiple "regions" and "locations", but the Progression region wil
...
+    if (chatipelago.isItemObtained(chatipelago.ITEMS.BUTTON_ACTIONVATION)) {
+        webhook.postInChat("Pressing the button");
+        chatipelago.claimCheck(chatipelago.LOCATIONS.BIG_RED_BUTTON);
+        chatipelago.goal();
+        goal = true;
+    }
+    else {
+        webhook.postInChat("You need the button activation");
+    }
+}*/
```
Comment (Dranzior) [inline bot-get.js:150]: Same as:
https://github.com/lallaria/Chatipelago/pull/2/files#r1826439955

**Review by Dranzior — APPROVED** [review]
Put small comment but nothing important, it does look good to me

**Review by lallaria — APPROVED** [review]

**Comment by lallaria** [comment]
committing

**Pull request merged**